editions/tw5.com/tiddlers/howtos/How to install TiddlyWiki5 on Heroku.tid

```diff
@@ -0,0 +1,49 @@
+created: 20141010110448509
+modified: 20141010110448509
+title: How to install TiddlyWiki5 on Heroku
+tags: [[TiddlyWiki on Node.js]]
+type: text/vnd.tiddlywiki
+
+1. Creating the local development environment
+
+npm install -g tiddlywiki # install tw5 globally
```

Comment:
These command line steps should be in a code block. Also the docs refer to "TiddlyWiki", not "tw5".